pkg/hintrunner/zero/zerohint_math.go

```diff
@@ -501,3 +502,99 @@ func createSplitIntHinter(resolver hintReferenceResolver) (hinter.Hinter, error)
 	}
 	return newSplitIntHint(output, value, base, bound), nil
 }
+
+func newSplitFeltHint(maxHigh, maxLow, low, high, value hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "SplitFelt",
+		Op: func(vm *VM.VirtualMachine, _ *hinter.HintRunnerContext) error {
+			//> from starkware.cairo.common.math_utils import assert_integer\nassert ids.MAX_HIGH < 2**128 and ids.MAX_LOW < 2**128\nassert PRIME - 1 == ids.MAX_HIGH * 2**128 + ids.MAX_LOW\nassert_integer(ids.value)\nids.low = ids.value & ((1 << 128) - 1)\nids.high = ids.value >> 128
```

Comment:
Could you actually use the enters instead of having `\n` in the string. It would be a lot more readable

Reply:
Modified the code, so that each line is in the new line.